service-providers/itering/25Q1-doc-edu-app-coordinator.md

```diff
@@ -0,0 +1,55 @@
+---
+applicant: Itering
+type: Docs & Education & App Coordinator
+timeline: 2025-Q1
+costs: 12,000 USDC
+payment: 0x0678E7cB36D3b9A591dF7E42CA23074aF977C52B (Arbitrum One)
+---
+
+## Team
+
+### Member
+
+- [Bear Wang](https://github.com/boundless-forest)
+- [Denny Wang](denny.wang@itering.io)
+- [Ran Ji](ran.ji@itering.io)
+
+### Contact
+
+- Contact name: Bear Wang
+- Contact email: bear.wang@itering.io
+
+### Legal Structure
+
+- Name: ITERING TECH PTE. LTD
+
+## Scope
+
+### Documentation
+
+High-quality documentation is crucial for the success of any project, as it helps attract new users and assists existing users in understanding the project better. This service will provide comprehensive documentation support for RingDAO, divided into two main categories:
+
+1. Official Documentation: This includes support for products within RingDAO, such as Darwinia docs, MsgPort docs, XAPI docs, website content, etc. These documents will be maintained long-term, updated according to product changes, and regularly polished.
+2. Community Documentation: This includes tutorials, monthly reports, and collaborative documents with other projects. These documents are essential and more flexible to accommodate community needs.
+
+### Community Education
+
+Building a strong and active community around RingDAO is essential. This service aims to bridge the gap between regular users and developers, ensuring a healthy and engaged community. The community education support includes:
+
+- Gathering and analyzing community feedback to improve products.
+- Collaborating with the marketing team and community members to organize online/offline events such as hackathons and AMAs.
+- Providing training and Q&A sessions for active community members to become community leaders, helping them organize and strengthen the community.
+
+### Application Coordination
+
+With the increasing number of applications built around RingDAO, it is crucial to ensure they are well-integrated, maintained, and updated. This service provides comprehensive application coordination support, including but not limited to:
+
+- ParaLink App: Launch the stable version of the ParaLink app and expand its user base from the RingDAO community to the broader Polkadot ecosystem.
+- RingDAO New Site: Oversee the development of the new RingDAO website, focusing on a refreshed look and improved user experience.
+- MsgPort New Site: Manage the development of the new MsgPort website, ensuring a modern design and enhanced user experience.
+- XAccount App: Coordinate and design the new version of the XAccount app.
```

Comment:
Please add apps.darwinia.network maintenance/management and keep updated:

https://apps.darwinia.network/
https://github.com/darwinia-network/apps